easybuild/easyconfigs/r/Ray-assembler/Ray-assembler-2.3.1-iimpi-2019a.eb

```diff
@@ -3,7 +3,7 @@
 
 easyblock = "MakeCp"
 
-name = "Ray"
+name = "Ray-assembler"
```

Comment:
please adapt `sources` to account for this change.

Reply:
Ah, thanks for the notice :)